animareflection/ui
# Migrate changesets docs

## Description

##### Task link: https://trello.com/c/DGEhGf73/213-migrate-changesets-docs-to-single-source-of-truth-docs-repo

**Related PRs (please review together):** https://github.com/animareflection/docs/pull/13, https://github.com/animareflection/docs-internal/pull/6

- Removed changesets docs in favor of organization docs for single source of truth

## Test Steps

- Verify docs link is correct (will only be live after https://github.com/animareflection/docs/pull/13 is merged)
